commands/audit/sca/conan/conan.go

```diff
@@ -0,0 +1,143 @@
+package conan
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/jfrog/gofrog/io"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	xrayUtils "github.com/jfrog/jfrog-client-go/xray/services/utils"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+
+	"github.com/jfrog/jfrog-client-go/utils/log"
+
+	"github.com/jfrog/jfrog-cli-security/utils"
+)
+
+const (
+	PackageTypeIdentifier = "conan://"
+)
+
+func BuildDependencyTree(params utils.AuditParams) (dependencyTrees []*xrayUtils.GraphNode, uniqueDeps []string, err error) {
+	// Prepare
+	currentDir, err := coreutils.GetWorkingDirectory()
+	if err != nil {
+		return
+	}
+	conanExecPath, err := getConanExecPath()
+	if err != nil {
+		return
+	}
+	// Build
+	return calculateDependencies(conanExecPath, currentDir, params)
+}
+
+func getConanExecPath() (conanExecPath string, err error) {
+	if conanExecPath, err = exec.LookPath("conan"); errorutils.CheckError(err) != nil {
+		return
+	}
+	if conanExecPath == "" {
+		err = errors.New("could not find the 'conan' executable in the system PATH")
+		return
+	}
+	log.Debug("Using Conan executable:", conanExecPath)
+	// Validate conan version command
+	version, err := getConanCmd(conanExecPath, "", "--version").RunWithOutput()
+	if errorutils.CheckError(err) != nil {
+		return
+	}
+	log.Debug("Conan version:", string(version))
```

Comment:
Can't see validation, if we are only supporting `v2`, please add validation for that and notify the user in case of error.
you can use `github.com/jfrog/gofrog/version`, see similar version compare usage in `commands/audit/sca/yarn/yarn.go:runYarnInstallAccordingToVersion`